### PR TITLE
Maven Integration

### PR DESCRIPTION
**Pull request by JACornelius**

### CODE REVIEW

**Comment by kev-chan** [comment]
add to gitignore
 .class
 twitter4j.properties
add instructions for how to generate a twitter4j properties file if one doesn't exist however

**Comment by kev-chan** [comment]
remove any unneeded checked in jar files and class files as well

**Comment by kev-chan** [comment]
remove target directory from checked in code and add to gitignore as well

**Comment by kev-chan** [comment]
please make the version of your app 1.0-SNAPSHOT

**Comment by kev-chan** [comment]
your groupId/artifactId should match your package/app name
also move the groupId/artifactId/version info to the top of the xml

**Comment by kev-chan** [comment]
put the twitter4j version inside a property in the pom file as well

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/target/twitter4j.properties
+/src/main/java/twitter4j.properties
+/src/main/java/TwitterApp.jar
+.DS-Store
+/bin
+/out
+/.idea
+/target
+/src/main/java/twitterapp/src/Main.class
```
Comment (kev-chan) [inline .gitignore]: you can use *.class

File: src/main/java/twitter4j.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 debug = true
-oauth.accessToken = 1009436937760006144-lMYADUmKvQShEMFByAQTsu11LfOKNP
-oauth.consumerSecret = R9UYKzyz7liKBdliAROoqw0uqI8d0H9IoMyV8R4OYRZIxzV8Dw
-oauth.consumerKey = gTFj02yhApZALpkmGukOc8UnN
-oauth.accessTokenSecret = RN51oxqnOAPH2mzS7LhkaKtfdxlwLe8ZlX65xihJpfci5
+oauth.accessToken =
```
Comment (kev-chan) [inline src/main/java/twitter4j.properties:2]: you may want to consider deleting the old keys and generating new ones so your keys don't make it out publicly

File: target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+twitterapp/src/Timeline.class
```
Comment (kev-chan) [inline target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst]: remove this and all of the other '.lst files"

File: target/maven-archiver/pom.properties [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#Created by Apache Maven 3.5.4
```
Comment (kev-chan) [inline target/maven-archiver/pom.properties]: this should not be checked in either

**Comment by kev-chan** [comment]
remove the checked in files from the target directory.  You may need to explicitly 'git add' them to make sure they get pushed to the remote branch.

**Review by kev-chan — CHANGES_REQUESTED** [review]
you probably need to update your pom.xml file since you rename a class

**Comment by JACornelius** [comment]
okay I think I completed all these changes you mentioned

**Pull request merged**